domains.txt

```diff
@@ -756,6 +845,7 @@ x4group.co.uk
 xactplacements.co.uk
 xcede.co.uk
 xcedesolutions.com
+xcm-uk.com
```

Comment:
XCM Rock
View 14 Jobs
XCM Rock Jobs Get Company Job Alerts
XCM UK, York House Road, Wetherby, York, YO267NH

Phone Us
xcmrock.gi
About Us
A recruitment consultancy with a focus on IT, Creative, CRM and Insight roles. Our staff have been selected not just because of their professionalism but because of their customer first ethos.

Its integrated partnerships we are interested in, not the quick wins of the smaller agencies, we position ourselves as the extension to your team, and act like it. We reward loyalty with unparalleled levels of service but also a fair fee policy.

https://www.cv-library.co.uk/list-jobs/296928/XCM-Rock